esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -3488,10 +3504,17 @@ bool CWsDfuEx::doLogicalFileSearch(IEspContext &context, IUserDescriptor* udesc,
     ForEach(*it)
         addToLogicalFileList(it->query(), NULL, version, logicalFiles);
 
-    if (version >= 1.24)
-        resp.setCacheHint(cacheHint);
+    if ((oldCacheHint != cacheHint) && (totalFiles > maxFiles))
```

Comment:
I will create a separate JIRA which needs UI changes from Migual and, possibly some WsSDFU changes from me, as soon as this PR is accepted. 

The warning should be stored (as cache hint) and shown on page >= 2.